tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -119,7 +119,7 @@ class TrtConversionParams(
         "rewriter_config_template", "max_workspace_size_bytes",
         "precision_mode", "minimum_segment_size", "is_dynamic_op",
         "maximum_cached_engines", "use_calibration", "max_batch_size",
-        "allow_build_at_runtime"
+        "allow_build_at_runtime", "allow_mixed_precision_on_native_ops"
```

Comment:
The meaning of native_ops is not clear. How about allow_low_precision_on_non_trt_converted_ops?

Reply:
How about `allow_mixed_precision_on_unconverted_ops` (see above for why I keep `mixed_precision` in the name)

Reply:
sure, then use allow_mixed_precision_on_unconverted_ops